project-base/app/src/FrontendApi/Component/SessionChecker/SessionChecker.php

```diff
@@ -15,7 +15,7 @@ public function onKernelResponse(ResponseEvent $event): void
     {
         $request = $event->getRequest();
 
-        if (!$request->hasSession() || !$request->getSession()->isStarted() || !str_contains($request->getRequestUri(), 'graphql')) {
+        if (!$request->hasSession() || !$request->getSession()->isStarted() || !str_contains($request->getRequestUri(), '/graphql/')) {
```

Comment:
GraphQL panel in the profiler does not work (http://127.0.0.1:8000/_profiler/xxxxxx?panel=graphql), due to this condition. 
This change was not necessary for this feature, but I have fixed it because I needed to check some graphql queries